app/javascript/gobierto_dashboards/webapp/components/TableRow.vue

```diff
@@ -36,10 +36,15 @@ export default {
     routingMember: {
       type: String,
       default: ''
+    },
+    routingAttribute: {
+      type: String,
+      default: 'id'
     }
   },
   created(){
     this.initFormattedItem();
+    this.routingId = this.item[this.routingAttribute];
```

Comment:
Initialize in data

Reply:
Please, install `eslint-plugin-vue`

![imagen](https://user-images.githubusercontent.com/817526/83168351-a603bc00-a111-11ea-8ffb-2c3847c3d15c.png)

Reply:
Installed. Yeah, there were a lot of warning, fixed most of them.